src/components/UI/ListItems/PlioListItem.vue

```diff
@@ -78,12 +78,8 @@ export default {
   props: {
     plioId: {
       // the id of the plio to render
-      default: "",
-      type: String,
-    },
-    org: {
-      // the active workspace
-      default: "",
+      // default is "dummy"
+      default: "dummy",
```

Comment:
why can't we make use of the pending variable only instead of adding this new dummy thing?

Reply:
will explain on call